api/processors/auxil/log_event_creator.py

```diff
@@ -26,13 +34,14 @@ def create(
                 )
                 for coordinator in group.coordinators.iterator()
             )
+        from_or_to_group_arg = {from_or_to_group.value: group}
 
         StudentLogEvent.objects.bulk_create(
-            StudentLogEvent(student=student, from_group=group, type=student_log_event_type)
+            StudentLogEvent(student=student, type=student_log_event_type, **from_or_to_group_arg)
             for student in group.students.iterator()
         )
 
         TeacherLogEvent.objects.bulk_create(
-            TeacherLogEvent(teacher=teacher, from_group=group, type=teacher_log_event_type)
+            TeacherLogEvent(teacher=teacher, type=teacher_log_event_type, **from_or_to_group_arg)
```

Comment:
Hm... I guess it was a mistake in one of the previous PRs, there shouldn't be just `from_group`, of course, there should be both.

Reply:
Fixed